styles/style.css

```diff
@@ -0,0 +1,152 @@
+body {
+    box-sizing: border-box;
+    background-color: #DCDCDC;
+}
+.header {
+    background-image: url("../images/header-image.jpg");
+    background-color: #003C63;
+    background-size: 100%;
+    height: 15rem;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: row;
```

Comment:
a justify content will be helpful here for positioning your elements. Play with the various values and see how they move around